molecule/default/molecule.yml

```diff
@@ -24,12 +24,14 @@ platforms:
     stop_signal: 'RTMIN+3'
     exposed_ports:
       - 8088/tcp
-      - 8081/tcp
+      - 8080/tcp
       - 5000/tcp
+      - 5555/tcp
     published_ports:
```

Comment:
Should we publish the minimal amount of ports possible?

Reply:
yeah, I've removed the unnecessary